### PR TITLE
Created list of allowed datatypes

### PR DESCRIPTION
**Pull request by guergana**
Not sure about the location. Put it in the root folder for now. Open to suggestions.

### CODE REVIEW

File: src/allowedDataTypes.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const allowedDataTypes = [
```
Comment (Ladsgroup) [inline src/allowedDataTypes.ts:1]: I'm pretty sure we don't yet support around of these data types like time or wikibase-item, they all need to be removed and only things that have "value type" of string should stay. I will find the list for you now.

**Review by Ladsgroup — CHANGES_REQUESTED** [review]

File: src/allowedDataTypes.ts (hunk 1 above)
Comment (Ladsgroup) [reply src/allowedDataTypes.ts:1]: The list: https://github.com/wikimedia/Wikibase/blob/master/lib/WikibaseLib.datatypes.php#L18

Comment (guergana) [reply src/allowedDataTypes.ts:1]: oh, ok. That was not made clear in the ticket, I can leave them commented out to be added later easily.

**Review by Ladsgroup — APPROVED** [review]
Thanks.

**Pull request merged**